.gitignore

```diff
@@ -22,6 +22,7 @@ parts/
 sdist/
 var/
 wheels/
+city_scrapers/get-pip.py
```

Comment:
Usually we'll want to keep unrelated .gitignore changes out of PRs. If you want to ignore something locally you can add it to `.git/info/exclude` in your repo